modules/nw-cluster-mtu-change-about.adoc

```diff
@@ -0,0 +1,81 @@
+
+[id="service-interruption-considerations_{context}"]
+== Service interruption considerations
+
+When you initiate an MTU change on your cluster the following effects might impact service availability:
+
+- At least two rolling reboots are required to complete the migration to a new MTU. During this time, some nodes are not available as they restart.
+
+// https://github.com/openshift/enhancements/pull/926/files#diff-166e452788d3711a8dfa7254d867f156c7b7c659f172e633acbf6852504ae358R180
+- Specific applications deployed to the cluster with shorter timeout intervals than the absolute TCP timeout interval might experience disruption during the change.
+
+// https://github.com/openshift/enhancements/pull/926/files#diff-166e452788d3711a8dfa7254d867f156c7b7c659f172e633acbf6852504ae358R190
+- Different endpoints in the cluster might use different MTUs during the migration. As a result it is possible that some network traffic is dropped until the MTU change is finalized.
+
+[id="mtu-value-selection_{context}"]
+== MTU value selection
+
+When planning your MTU migration there are two related but distinct MTU values to consider.
+
+* *Hardware MTU*: This MTU value is set based on the specifics of your network infrastructure.
+* *Cluster network MTU*: This MTU value is always less than your hardware MTU to account for the cluster network overlay overhead. The specific overhead is determined by your cluster network provider:
+** *OVN-Kubernetes*: `100` bytes
+** *OpenShift SDN*: `50` bytes
+
+// modules/nw-operator-cr.adoc
+If your cluster requires different MTU values for different nodes, you must set this value to `100` less than the lowest MTU value in your cluster. For example, if some nodes in your cluster have an MTU of `9001`, and some have an MTU of `1500`, you must set this value to `1400`.
+
+[id="how-the-migration-process-works_{context}"]
+== How the migration process works
+
+The following table summarizes the migration process by segmenting between the user-initiated steps in the process and the actions that the migration performs in response.
+
+.Live migration of the cluster MTU
+[cols="1a,1a",options="header"]
+|===
+
+|User-initiated steps|{product-title} activity
+
+|
+Set the following values in the Cluster Network Operator configuration:
+
+- `spec.migration.mtu.network.from`
+- `spec.migration.mtu.network.to`
+- `spec.migration.mtu.machine.to`
+
+|
+*Cluster Network Operator (CNO)*: Confirms that each field is set to a valid value.
+
+- The `mtu.network.from` field must equal `network.status.clusterNetworkMTU` which is the current MTU of the cluster network.
+- The `mtu.network.to` field must allow for the overlay overhead for the cluster network provider. For OVN-Kubernetes, this value is `100` and for OpenShift SDN this value is `50`. The value of `mtu.network.to` can equal the value of `mtu.network.from`.
+
```

Comment:
@jcaamano, I thought it was `network.{from,to}` that reflects the value for the cluster network provider? And that this value needs to accommodate the SDN overhead? So that this is set relative to the network link MTU?

Reply:
You are right. I was getting confused because we are mixing the term `cluster network` and `cluster network provider`. Previously in the document you define `Cluster network MTU` and `Hardware MTU` so we should probably stick to those terms to avoid confusion.

I think this does not make sense
`The value of mtu.network.to can equal the value of mtu.network.from`
Although technically correct, it does not make a lot of sense if the purpose of the instruction is actually to change the cluster network MTU. I did give a comment about `mtu.machine.to` in this regard before but not about ` mtu.network.to`

Overall, I might rephrase to something more like:
```
- The `mtu.machine.to` must be set to the a desired Hardware MTU or to the current Hardware MTU. Note that this value is only informative to the MTU migration procedure and that the Hardware MTU has to be configured by other means. Some suggestion are provided later on.
- The `mtu.network.from` field must equal `network.status.clusterNetworkMTU` which is the current MTU of the cluster network.
- The `mtu.network.to` field must be set to the new desired cluster network MTU and must be lower than the Hardware MTU to allow for the overlay overhead of the cluster network provider. For OVN-Kubernetes, the overhead is `100` and for OpenShift SDN this overhead is `50`. 
```

Reply:
I still miss something here that at least conveys the idea that `mtu.machine.to` must be set to the Hardware MTU in a similar way as I phrased above.

Reply:
@jcaamano, I'll look, thanks!

Reply:
@jcaamano, updated, can you confirm? Thanks!